### PR TITLE
Version1 adding .gitignore and deleting ViewModels and Models folder

### PR DESCRIPTION
**Pull request by alex308248**
Trying to fix issue #2 and #3. Deleted the extra folder and add a raw .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
...
### Xcode Patch ###
*.xcodeproj/*
!*.xcodeproj/project.pbxproj
!*.xcodeproj/xcshareddata/
!*.xcodeproj/project.xcworkspace/
!*.xcworkspace/contents.xcworkspacedata
/*.gcno
**/xcshareddata/WorkspaceSettings.xcsettings

# End of https://www.toptal.com/developers/gitignore/api/macos,xcode,swift
```
Comment (YLee2121) [inline .gitignore]: Is it a .gitignore but compressed into one line?? 
Will it be a normal file after PR to the repo??

Comment (alex308248) [reply .gitignore]: Fixed. It happened due to the difference of newline character in Windows and Mac system.

Comment (YLee2121) [reply .gitignore]: OK Done reviewing!!!!
Nice Work!

**Review by YLee2121 — APPROVED** [review]
Done Review!

**Review by YLee2121 — APPROVED** [review]
Done Review!

**Review by cheniwei — APPROVED** [review]

**Pull request merged**